changelogs/unreleased/ghe-523-max-config-param-length.md

```diff
@@ -0,0 +1,4 @@
+## feature/app
```

Comment:
It's rather `feature/box`, not `feature/app`. Also, let's not refer to the EE bug because CE users don't have to it. It's okay to leave this entry without a ticket reference.

Reply:
Thanks, fixed.

Reply:
You forgot to drop the ticket reference.

Reply:
Also, `box.cfg{}` should be enclosed in backticks (\`) because it's a function name.

Reply:
True, thanks, fixed.